config.py

```diff
@@ -545,7 +545,8 @@ def site_configuration_last_update(cls, _db, known_value=None,
         if not known_value:
             from model import Timestamp
             known_value = Timestamp.value(
-                _db, cls.SITE_CONFIGURATION_CHANGED, None
+                _db, cls.SITE_CONFIGURATION_CHANGED, service_type=None,
+                collection=None
```

Comment:
This is an internal process, not one of the three recognized service types, so we leave `service_type` null.